content/ecosystem/data.toml

```diff
@@ -1342,4 +1347,4 @@ categories = ["tools"]
 [[items]]
 name = "ziyasal/natura"
 source = "github"
-categories = ["animation"]
+categories = ["animation"]
```

Comment:
Can you add the end-of-file blank line back?

Reply:
Done